src/execution/mod.rs

```diff
@@ -42,6 +62,14 @@ impl AdvanceTo {
     }
 }
 
+/// Emit the supplied item into the dataflow at the current epoch
+///
+/// Emit is how we introduce input into a dataflow:
+///
+/// >>> def input_builder(worker_index, worker_count):
+/// ...     for i in range(10):
+/// ...     yield AdvanceTo(i) # Advances the epoch to i
```

Comment:
this indentation looks off here and above, like the yields should be tabbed one more